jar-infer/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -83,22 +81,13 @@ private void testTemplate(
   private void testJARTemplate(
       String testName,
       String pkg, // in dot syntax
-      String jarPath, // in dot syntax
-      String expected // expected astubx MD5 hash
+      String jarPath // in dot syntax
       ) {
     DefinitelyDerefedParamsDriver defDerefParamDriver = new DefinitelyDerefedParamsDriver();
     try {
       defDerefParamDriver.run(jarPath, "L" + pkg.replaceAll("\\.", "/"));
-      String aStubXPath =
-          jarPath.substring(0, jarPath.lastIndexOf('.'))
-              + "/META-INF/nullaway/"
-              + pkg.replaceAll("/", "\\.")
-              + ".astubx";
-      Assert.assertTrue("astubx file not found! - " + aStubXPath, new File(aStubXPath).exists());
-      String hash =
-          DatatypeConverter.printHexBinary(
-              MessageDigest.getInstance("MD5").digest(Files.readAllBytes(Paths.get(aStubXPath))));
-      Assert.assertEquals("astubx MD5 checksum mismatch!", expected, hash);
```

Comment:
Was there an unexpected problem with doing the `MD5` checksum test? Too frequent changes or anything like that? I am fine with removing it if it's flaky, I just would like to know why.

Reply:
The MD5 hash can be checked for the astubx file alone, but packing a JAR changes the hash every time due to updated timestamps.

Reply:
Makes sense to me then.